L1Trigger/L1THGCal/interface/HGCalTriggerFECodecBase.h

```diff
@@ -99,7 +101,7 @@ namespace HGCalTriggerFE {
           << "Data payload was already set for HGCTriggerFECodec: "
           << this->name() << " overwriting current data!";
       }
-      static_cast<Impl&>(*this).setDataPayloadImpl(mod,ee,fh,bh);
+      static_cast<Impl&>(*this).setDataPayloadImpl(mod,geom,ee,fh,bh);
```

Comment:
Indeed, you are correct! However, I think the correct action is to fix the geometry class itself rather than kludge things downstream of it. I'll send you a pull request in a bit.